Python/navio/Navio2/RCInput.py

```diff
@@ -0,0 +1,20 @@
+class RCInput:
+    CHANNEL_COUNT = 14
+    channels = []
+
+    def __init__(self):
+        for i in range(0, self.CHANNEL_COUNT):
+            try:
+                f = open("/sys/kernel/rcio/rcin/ch%d" % i, "r")
+                self.channels.append(f)
+            except:
+                print("Can't open file /sys/kernel/rcio/rcin/ch%d" % i)
+
+    def read(self, ch):
+        try:
+            value = self.channels[ch].read()
+            position = self.channels[ch].seek(0, 0)
+            return value[:-1]
+        except IndexError:
+            print("Channel number too large")
+            exit(1)
```

Comment:
It's a good practice to throw exception on higher level instead of exiting inside utility method